### PR TITLE
Feat/gffread nf tests

### PR DESCRIPTION
**Pull request by kevinpryan**
## PR checklist

Linked to #3935

- [x] Added `nf-test` and snapshot for `gffread`
- [x] Added tags.yml for `gffread` `nf-test`
- [x] Removed `gffread` from `tests/config/pytest_modules.yml`
- [x] Worked locally on docker, singularity and conda
- [ ] If you've fixed a bug or added code that should be tested, add tests!

Linting: Fails 1 Module Test - `missing entry in pytest_modules.yml` due to removing gffread from `pytest_modules.yml`

### CODE REVIEW

**Comment by kevinpryan** [comment]
`nf-test` checks failing - error at `Install nf-tests` step

File: modules/nf-core/gffread/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    name "Test Process GFFREAD"
+    script "../main.nf"
+    process "GFFREAD"
+    tag "gffread"
+    tag "modules_nfcore"
+<<<<<<< HEAD
+    tag "modules"
+=======
+>>>>>>> 5b289170dd3d7d594bdb72224cb65ed1fb7bc22f
```
Comment (maxulysse) [inline modules/nf-core/gffread/tests/main.nf.test:11]:
```suggestion
    tag "modules"
```

**Review by maxulysse — COMMENTED** [review]
Looking nice

**Comment by maxulysse** [comment]
Fixed some merging conflicts and tags related isues

**Pull request merged**